sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestSuites.java

```diff
@@ -19,56 +19,62 @@
  */
 package org.sonar.plugins.php.phpunit.xml;
 
+import com.google.common.base.Objects;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.base.Objects'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)